services/backend/Gemfile

```diff
@@ -33,11 +33,11 @@ group :development, :test do
   gem 'rspec_junit_formatter'
 
   # monitoring
-  gem 'bullet'
-  gem 'rack-mini-profiler', require: false
-  gem 'flamegraph'
-  gem 'stackprof'
-  gem 'memory_profiler'
+  # gem 'bullet'
+  # gem 'rack-mini-profiler', require: false
+  # gem 'flamegraph'
+  # gem 'stackprof'
+  # gem 'memory_profiler'
```

Comment:
I think you should remove the comments, and move the ddtrace gem under `# monitoring`